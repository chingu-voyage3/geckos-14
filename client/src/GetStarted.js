import React, { Component } from 'react';

class GetStarted extends Component {
  render() {
    return (
      <div>GetStarted Page</div>
    );
  }
}

export default GetStarted;
