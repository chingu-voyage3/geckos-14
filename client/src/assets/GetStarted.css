.getstarted {
  display: grid;
  grid-template-columns: 1fr 3fr 1fr;
}
.getstarted-sidebar {
  margin-top: 50px;
  margin-left: 20px;
}
.getstarted-content {
}
.getstarted-content-item {
  margin: 50px 0 50px 0;
}
