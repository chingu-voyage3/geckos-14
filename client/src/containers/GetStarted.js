import React, { Component } from 'react';
import SideBar from '../components/getstarted/SideBar';
import Content from '../components/getstarted/Content';
import * as d from '../tempData';

class GetStarted extends Component {
  render() {
    return (
      <div>
        <SideBar items={d.sideBarItems} />
        <Content content={d.getStartedContent} />
      </div>
    );
  }
}

export default GetStarted;
