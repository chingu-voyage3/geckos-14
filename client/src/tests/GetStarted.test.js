import React from 'react';
import GetStarted from '../containers/GetStarted';

import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const wrapper = shallow(<GetStarted />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
