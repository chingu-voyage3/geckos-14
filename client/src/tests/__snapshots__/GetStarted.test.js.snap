// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <h2>
    Let's Get You Started with these basics steps
  </h2>
  <ul>
    <li>
      Step 1
    </li>
    <li>
      Step 2
    </li>
    <li>
      Step 3
    </li>
  </ul>
</div>
`;
